lms/envs/common.py

```diff
@@ -5447,7 +5456,12 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
 def _should_send_certificate_events(settings):
     return settings.FEATURES['SEND_LEARNING_CERTIFICATE_LIFECYCLE_EVENTS_TO_BUS']
 
+
 #### Event bus producing ####
+
+def _should_send_learning_badge_events(settings):
+    return settings.FEATURES['BADGES_ENABLED']
```

Comment:
1. Will this toggle be used for anything other than `_should_send_learning_badge_events`? It seems like the toggle name could be much more descriptive, because this seems like there is a whole new feature that might be enabled/disabled.
2. Is there any relationship with the badges DEPR: https://github.com/openedx/edx-platform/issues/31541? If not, you can see how a toggle named `FEATURES['BADGES_ENABLED']` would confuse me to thinking there would be.

Reply:
@robrap: thank you for raising this. 

@GlugovGrGlib: Could you help us out here? If anything needs to be changed, we can do it in a follow-up PR and backport it to Redwood.

Reply:
@GlugovGrGlib: since the redwood branch is created now off https://github.com/openedx/edx-platform/tree/2u/release, these changes didn't make it in time. So if this needs to land in redwood, you folks still need to open a PR against open-release/redwood.master

Reply:
@robrap
1. This is the first phase of the badging initiative outlined in the https://github.com/openedx/platform-roadmap/issues/280, which is expected to include additional milestones in the future. These potential milestones may encompass issuing badges for various achievements such as passing parts of the courses, and displaying claimed badges alongside courseware. The intention is to avoid embedding specific badging logic directly within the edx-platform. Instead, the plan is to emit generic events in the LMS or CMS, which can then be consumed and processed by the credentials IDA.

2. From a product perspective, this feature represents a relaunch of the legacy open badges functionality, which was deprecated last year. Although it serves similar use cases, the current solution centralizes all processing logic within the credentials IDA. Additionally, future plans include support for multiple badging services beyond Credly, such as Accredible or others. Using a generic name like `BADGES_ENABLED` can ease the adoption of the feature across different community releases. With each release, users won't need to reconfigure LMS to use new functionality introduced in future milestones. At the same time, the primary configuration for processing remains within the credentials IDA.

Reply:
@mariajgrimaldi sure will do, thanks